dspace-server-webapp/src/main/java/org/dspace/app/rest/model/hateoas/HarvestedCollectionResource.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model.hateoas;
+
+import org.dspace.app.rest.model.HarvestedCollectionRest;
+import org.dspace.app.rest.model.HarvesterMetadataRest;
+import org.dspace.app.rest.utils.Utils;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * HarvestedCollection Rest HAL Resource. The HAL Resource wraps the REST Resource
+ * adding support for the links and embedded resources
+ *
+ * @author Jelle Pelgrims (jelle.pelgrims at atmire.com)
+ */
+public class HarvestedCollectionResource extends HALResource<HarvestedCollectionRest> {
+
+    @Autowired
+    private Utils utils;
+
+    public HarvestedCollectionResource(HarvestedCollectionRest data) {
+        super(data);
+        embedResource("metadata_configs", data.getMetadataConfigs());
```

Comment:
I believe this should be changed from "metadata_configs" to "harvestermetadata" per discussion in the REST contract here: https://github.com/DSpace/Rest7Contract/pull/66/files#r308300750